opendevin/controller/state/state.py

```diff
@@ -37,6 +50,7 @@ class State:
     error: str | None = None
     agent_state: AgentState = AgentState.LOADING
     resume_state: AgentState | None = None
+    traffic_control_state: TRAFFIC_CONTROL_STATE = TRAFFIC_CONTROL_STATE.NORMAL
```

Comment:
Nice! Just a small curiosity, does it also resume normally and what means 'normally' here, paused? I mean, when the user quits, then restarts in the browser, and is asked whether to resume the old session.

Reply:
`NORMAL` means the traffic control mechanism is functioning normally and we haven't hit the limit yet. I am horrible at naming so any suggestion on renaming is welcome.

Regarding resuming old session... I've never tried that tbh, but let me try.

Reply:
Seems it works smoothly, if that's what you are asking:

1. Open OpenDevin and ask it to write a python game
2. It does something and then hits the step limit
3. I refresh the webpage and choose to "resume old session"
4. I click on <img width="133" alt="image" src="https://github.com/OpenDevin/OpenDevin/assets/25746010/4b34711e-7169-487c-82e2-f32ff784ccef"> button and then OpenDevin continue